parla/cuda.py

```diff
@@ -50,13 +52,28 @@ class _GPUMemory(Memory):
     def np(self):
         return _DeviceCUPy(self.device)
 
+    def asarray_sync(self, src):
```

Comment:
I think you mean for this to be asarray_async? sync refers to the cuda semantics that we're trying to avoid.

Reply:
Yes my bad. I put synchronize() before right after the copy. That's why I put that name. I am going to fix this name.

Reply:
Fixed